1.5/Source/PawnStorages/PawnStoragesModule-VEF/HarmonyPatches/CompPawnStorageNutrition_Patches.cs

```diff
@@ -0,0 +1,70 @@
+﻿using HarmonyLib;
+using PipeSystem;
+using RimWorld;
+using UnityEngine;
+
+namespace PawnStorages.VEF.HarmonyPatches;
+
+[HarmonyPatch(typeof(CompPawnStorageNutrition))]
```

Comment:
I don't understand this class at all. Why use harmony on our own mod, just subclass? Harmony patches introduce overhead, it's small but still, this is a perf mod, using the lower perf option on our own class feels odd. I'd understand if it was a way to convert old to new so it was easy to kill later or something. If it's just about cleanly adding the code for the paste without affecting the base then using an xml patch to swap out the class might be better? Or having a PawnStorageNutritionSource interface and having that be swapped out instead of the whole class. Idk if you think it's called infrequently enough I'm not super opposed as long as there's a cache.